bolt.rb

```diff
@@ -1,19 +1,46 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
 
 require 'bolt/inventory'
-# Remove after next Bolt release.
-require 'bolt/logger'
 require 'wash'
 
+# For now we're going to mock out the plugin interface. It requires more setup
+# to get PAL/PuppetDB working.
+class Plugin
+  attr_accessor :plugin_hooks
+  def initialize
+    @plugin_hooks = {}
+  end
+
+  def reference?(_input)
+    false
+  end
+
+  def resolve_references(ref)
+    ref
+  end
+
+  def resolve_top_level_references(ref)
+    ref
+  end
+end
+
 class Boltwash < Wash::Entry
   label 'bolt'
   is_singleton
   parent_of 'Group'
+  description <<~DESC
+    A plugin for Puppet Bolt's inventory. You can see target configuration in entry
+    metadata. You can SSH to targets and view their filesystems in 'fs' directories.
```

Comment:
You mean the `meta` command? Also, looks like `docs` doesn't mention `meta` unless there's at least one supported attribute. We should probably have a different message if the entry has no attributes, but has some partial metadata. Filed https://github.com/puppetlabs/wash/issues/697 to fix this.

Reply:
Yes, wasn't sure a better way to phrase it.